docs/user/advanced_features.rst

```diff
@@ -1243,6 +1244,50 @@ computation time and complex setup.
 	| Avg Mechanical Power (kW)  |      N/A     |    300      |    241     |
 	+----------------------------+--------------+-------------+------------+
 
+.. _control-reactive-with-PTO:
+
+Reactive Control with Direct Drive Power Take-Off
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The previous controllers only considered the mechanical power output. Although maximization
+of mechanical power allows for the maximum energy transfer from waves to body, it often does 
+not lead to maximum electrical power. The previous controller examples demonstrate the 
+controller types and energy transfer from waves to body, but the important consideration of 
+electrical power requires a PTO model. This example applies a reactive controller to the 
+sphere body with a simplified direct drive PTO model to maximize electrical power. Within 
+the Simulink subsystem for determining the PTO force, the controller prescribes the ideal or 
+desired force which is fed into the direct drive PTO. The current in the generator is then 
+used to control the applied force. 
+
+.. figure:: /_static/images/piPTOSimulink.png
+   :width: 500pt 
+   :align: center
+
+The PTO parameters used for this example are defined in the wecSimInputFile and correspond to 
```

Comment:
Editorial comment: can we use `wecSimInputFile.m` instead of the current labeling to assist in defining there is a separate file that can be referenced. Also not possible now, but can we add a citation, link, or point to the soon to be included applications case?

Reply:
First part resolved. Leaving open to add link to point to applications case when possible.

Reply:
Thanks.